core/src/main/java/hudson/model/AbstractCIBase.java

```diff
@@ -158,21 +160,31 @@ private void updateComputer(Node n, Map<String, Computer> byNameMap, Set<Compute
     @CheckForNull
     private Computer createNewComputerForNode(Node n, boolean automaticAgentLaunch) {
         Computer c = null;
-        Map<Node, Computer> computers = getComputerMap();
+        ConcurrentMap<Node, Computer> computers = getComputerMap();
```

Comment:
This change is good. But the type of the map is still `Map` rather than `ConcurrentMap` in the calls to `#getComputerMap` in `AbstractCIBase#getComputer` and `AbstractCIBase#updateComputerList` further below, which is inconsistent. Can you update those to `ConcurrentMap` for consistency?